Objects/regions.c

```diff
@@ -480,4 +480,221 @@ PyObject* _Py_MakeImmutable(PyObject* obj)
     _Py_VPYDBG("<< makeimmutable complete\n\n");
 
     return obj;
-}
\ No newline at end of file
+}
+
+typedef struct RegionObject RegionObject;
+typedef struct RegionMetadata RegionMetadata;
+
+struct RegionMetadata {
```

Comment:
This internal types are typically lowercase with no spacing (sic).

```suggestion
struct regionmetadata {
```

And this would need passing through the rest of the function names.